spec/models/article_spec.rb

```diff
@@ -155,9 +155,9 @@ def build_and_validate_article(*args)
     describe "#main_image_background_hex_color" do
       it "must have true hex for image background" do
         article.main_image_background_hex_color = "hello"
-        expect(article.valid?).to eq(false)
```

Comment:
Everything else in this PR was auto-corrected when I merged `main` into my branch 🤷‍♀️